src/deploy/osp_deployer/settings_sanity.py

```diff
@@ -585,3 +585,11 @@ def verify_sriov_dependencies(self):
             raise AssertionError("SR-IOV and DVR can not be " +
                                  "enabled together.Please disable" +
                                  "one feature and verify the settings.")
+
+    def verify_hw_offload_dependencies(self):
+        logger.debug("verifying Smart NIC is enabled when SR-IOV is enabled")
```

Comment:
nit: change the above log message to:
Verifying SR-IOV is enabled when Smart NIC is enabled